src/picterra/tracer_client.py

```diff
@@ -288,3 +287,131 @@ def list_plots_analyses(
         if page_number is not None:
             data["page_number"] = int(page_number)
         return self._return_results_page(f"plots_groups/{plots_group_id}/analysis/", data)
+
+    def list_plots_analysis_reports(
+        self,
+        plots_group_id: str,
+        plots_analysis_id: str,
+    ):
+        """
+        List all the reports belonging to a given plots analysis
+
+        Args:
+            plots_group_id: id of the plots group on which we want to list the analyses
+            plots_analysis_id: id of the plots analysis for which we want to list the reports
+
+        Returns:
+            ResultsPage: A ResultsPage object that contains a slice of the list
+                of plots analyses reports dictionaries
+        """
+        return self._return_results_page(
+            f"plots_groups/{plots_group_id}/analysis/{plots_analysis_id}/reports/"
+        )
+
+    def list_plots_analyses_report_types(
+        self,
+        plots_group_id: str,
+        plots_analysis_id: str
+    ):
+        """
+        List all the plots analyses report types the user can use (see create_plots_analysis_report)
+
+        Args:
+            plots_group_id: id of the plots group
+            plots_analysis_id: id of the plots analysis
+
+        Returns:
+            list[str]: List of the possible report types ("name" and "report_type")
+        """
+        resp = self.sess.get(
+            f"plots_groups/{plots_group_id}/analysis/{plots_analysis_id}/reports/types/"
```

Comment:
There is a `_full_url` missing here (I'm getting a `MissingSchemaError` when calling this function)

Reply:
I think the slightly confusing this is that you don't need this when calling `_return_results_page` as it's added automatically there